mobile/src/main/java/rs/readahead/washington/mobile/views/adapters/reports/ReportsFilesRecyclerViewAdapter.kt

```diff
@@ -104,13 +98,7 @@ open class ReportsFilesRecyclerViewAdapter(
                 if (isImageFileType(vaultFile.mimeType)) {
                     this.showImageInfo(vaultFile)
                     Glide.with(context)
```

Comment:
For kotlin class let's use an extension function it will make it more easier `fun ImageView.loadImage(thumb: ByteArray){
    Glide.with(this)
        .load(thumb)
        .diskCacheStrategy(DiskCacheStrategy.NONE)
        .skipMemoryCache(true)
        .into(this)
}`